### PR TITLE
Add Flattr profile

### PR DESCRIPTION
**Pull request by CCoffie**

### CODE REVIEW

File: chapters/bizleg-case-studies/crowdfunding/Flattr.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+What drew you into wanting to know more about the organization?
+How did you/your group decide on and agree?
+
+Flattr was company that we had never heard of before and wanted to learn more
+about the platform.
+
+### Organizational Details
+
+1. Is the subject of your profile a corporate entity?
+  - Flattr is the corporate entity
```
Comment (ryansb) [inline chapters/bizleg-case-studies/crowdfunding/Flattr.md]: Can you change these to match the format:

> the question ?

Your answer.

File: chapters/bizleg-case-studies/crowdfunding/Flattr.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  - None
+
+### Community Architecture
+Your subject likely runs or contributes to one or more Open Source products or
+projects. Choose one (or more) of these and answer the following questions:
+
+If applicable, list and provide links to:
+
+1. The project's IRC Channel
+  - #flattr on freenode with a max of 5 users in the past year
```
Comment (qwertos) [inline chapters/bizleg-case-studies/crowdfunding/Flattr.md]: Please escape the `#` in the channel name.

**Comment by CCoffie** [comment]
The suggested changes have been made.

**Comment by qwertos** [comment]
Thanks Merged :smile:

**Pull request merged**